pubspec.yaml

```diff
@@ -4,27 +4,29 @@ version: 0.12.0
 repository: https://github.com/johnpryan/flutter_map
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
   flutter: ">=2.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  tuple: ^1.0.2
-  latlong: ^0.6.1
+  tuple: ^2.0.0
+  latlong2: ^0.8.0
   positioned_tap_detector_2: ^1.0.0
-  transparent_image: ^1.0.0
+  transparent_image: ^2.0.0
   async: ^2.1.0
-  flutter_image: ^3.0.0
-  vector_math: ^2.0.0
+  flutter_image:
```

Comment:
I think, the `git:` part should be declared using a `dependency_override`

Reply:
Why?

Reply:
I don't know exactly what the convention is for this repositoy, but I think, it might be better to show that this is a temporary workaround until the changes are uploaded to `pub`

Reply:
I see it differently. A `dependency_override` can easily be missed to be removed when the time comes. Also we don't have to override the dependency of an indirect dependency in this case

Reply:
I do think it's easier to remove dependency overrides when it's time to use the published version.